### PR TITLE
docs(changelog): version 1.3.0 [citest skip]

### PR DESCRIPTION
**Pull request by spetrosi**
Update changelog and .README.html for version 1.3.0

Signed-off-by: Sergei Petrosian <spetrosi@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+
+### New Features
+
+- feat: Add options for rate limit interval and burst (#64)
+
+  Enhancement:
+  Add two new variables, allowing the rate limit to be configured using the role.
+  
+  Reason:
+  IHAC which needs these settings to be changed from their default values, and they would like to use RHEL System Roles as much as possible.
```
Comment (ephracis) [inline CHANGELOG.md:15]: I think lines 10 to 15 should be removed here as it doesn't sound right for a change log entry.

When I filled out the pull request template I wasn't aware that the body would be included in the change log, so now it comes out a bit weird (especially the "Reason" part).

Comment (richm) [reply CHANGELOG.md:15]: Well, we have automation that just automatically builds the changelog entries from the PR.
I suppose we could require two additional doc fragments when submitting a PR - one for the PR body with the details and the reason, and one for the changelog snippet, but that seems like too much overhead.
I agree in this case that it doesn't really add anything to the changelog

Comment (richm) [inline CHANGELOG.md:15]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+[1.3.0] - 2024-04-02
+--------------------
+
+### New Features
+
+- feat: Add options for rate limit interval and burst (#64)
+
+  Enhancement:
+  Add two new variables, allowing the rate limit to be configured using the role.
+
```
Comment (richm) [inline CHANGELOG.md:13]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+  Enhancement:
+  Add two new variables, allowing the rate limit to be configured using the role.
+  
+  Reason:
+  IHAC which needs these settings to be changed from their default values, and they would like to use RHEL System Roles as much as possible.
+
+### Other Changes
+
+- ci: fix python unit test - copy pytest config to tests/unit (#62)
+
```
Comment (richm) [inline CHANGELOG.md:20]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+
```
Comment (richm) [inline CHANGELOG.md:21]:
```suggestion
```

File: CHANGELOG.md (hunk 1 above)
Comment (spetrosi) [reply CHANGELOG.md:13]: @ephracis please modify PR description in #64 so it has a text suitable for a changelog entry?

Comment (ephracis) [reply CHANGELOG.md:15]: @spetrosi done! :)

**Review by richm — APPROVED** [review]

**Pull request merged**